frontend/src/pages/devices/List.tsx

```diff
@@ -153,6 +155,16 @@ export const List = () => {
         return <>{ level }%</>;
     };
 
+    const renderOnline = (entity: DeviceT) => {
+        if (entity.rawDevice.status === "ONLINE") {
```

Comment:
Note with your current logic that the `else` will never be hit (it's either `=== "ONLINE` or `!== "ONLINE`, there's no third option).

I think you can simplify your logic to something like:
```js
if (entity..... "ONLINE") {
  return whatever;
}
return theOtherThing;
```